code/modules/atmospherics/environmental/LINDA_fire.dm

```diff
@@ -5,11 +5,11 @@
 
 
 
-/turf/proc/hotspot_expose(exposed_temperature, exposed_volume, soh = 0)
+/turf/proc/hotspot_expose(exposed_temperature, exposed_volume, soh = 0, heatless = 0)
```

Comment:
no need for = 0 for either

Reply:
I was emulating the 'soh' variable, looks like that just controls whether it creates fires by itself; should that follow the same rules?

Reply:
I assumed that since soh is defined as 0 by default, that this would need to be as well.